feature-configs/cn-ran-overlays/all-in-one/example/sriov/sriov-network-node-policy-netdevice-mh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mh-dpdk-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
```

Comment:
This will work with intel only.
Let's document it.
Mellanox must be netdevice + isrdma = true.

Reply:
@josephdrichard

Reply:
This is targeting a specific hardware config, in a lab that it's using intel.   Where would you suggest documenting this?

Reply:
A note in the docs would work I guess. Something like "sriov configuration notes".
On top of that, a thing you can do is to add a parallel sriov configuration for mellanox, with the same resource name.
At that point, the device plugin will discover only the available ones and will put the capacity on the node accordingly.

It would help us in deploying this on clusters where we have mellanox too.
Finally, I'd document the fact that this configuration depends on the name of the pfs as well, which may differ from cluster to cluster.

Reply:
@fedepaol I will add this to the README.md